src/qibocal/protocols/characterization/__init__.py

```diff
@@ -20,10 +20,12 @@
 from .dispersive_shift_qutrit import dispersive_shift_qutrit
 from .fast_reset.fast_reset import fast_reset
 from .flipping import flipping
-from .flux_dependence.qubit_crosstalk import qubit_crosstalk
-from .flux_dependence.qubit_flux_dependence import qubit_flux
-from .flux_dependence.resonator_crosstalk import resonator_crosstalk
-from .flux_dependence.resonator_flux_dependence import resonator_flux
+from .flux_dependence.qubit_flux_dependence import qubit_crosstalk, qubit_flux
+from .flux_dependence.qubit_flux_tracking import qubit_flux_tracking
+from .flux_dependence.resonator_flux_dependence import (
+    resonator_crosstalk,
+    resonator_flux,
+)
```

Comment:
```suggestion
from .flux_dependence.qubit_flux_tracking import qubit_flux_tracking
from .flux_dependence.qubit_crosstalk import qubit_crosstalk
from .flux_dependence.qubit_flux_dependence import qubit_flux
from .flux_dependence.resonator_crosstalk import resonator_crosstalk
from .flux_dependence.resonator_flux_dependence import resonator_flux
```
This should fix tests

Reply:
@andrea-pasquale @Edoardo-Pedicillo the PR is ready to be merged